src/main/java/com/github/stephengold/joltjni/Triangle.java

```diff
@@ -29,7 +30,7 @@ of this software and associated documentation files (the "Software"), to deal
  *
  * @author Stephen Gold sgold@sonic.net
  */
-final public class Triangle extends JoltPhysicsObject {
+final public class Triangle extends JoltPhysicsObject implements ConstTriangle {
```

Comment:
Now that `ConstTriangle` interface exists, the public `getMaterialIndex()` , `getUserData()`, and `putVertices()` methods should be moved to a new section titled "ConstTriangle methods".